src/netex_france/exporter.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2017 Kisio Digital and/or its affiliates.
+//
+// This program is free software: you can redistribute it and/or modify it
+// under the terms of the GNU Affero General Public License as published by the
+// Free Software Foundation, version 3.
+
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+// FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+// details.
+
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>
+
+//! Exporter for Netex France profile
+use crate::{model::Model, Result};
+use std::{convert::AsRef, path::Path};
+
+/// Struct that can write an export of Netex France profile from a Model
+pub struct Exporter<P>
+where
+    P: AsRef<Path>,
+{
+    model: Model,
+    path: P,
+}
+
+impl<P> Exporter<P>
+where
+    P: AsRef<Path>,
+{
+    /// Build a Netex France profile exporter from the model.
+    /// `path` is the expected output Path where the Netex France is going to be
+    /// written. It should be a folder that already exists.
+    pub fn new(model: Model, path: P) -> Self {
+        Exporter { model, path }
+    }
+
+    /// Actually write `model` into `path` as a Netex France profile.
+    pub fn write(&self) -> Result<()> {
```

Comment:
I prefer `write(&self, path: P)` if path is not used elsewhere

Reply:
Good point!